debian/service

```diff
@@ -0,0 +1,11 @@
+[Unit]
```

Comment:
Does this need to be duplicated?

Reply:
Ubuntu/Debian and RH use different conventions on the prometheus stuff which includes naming. I tried to follow them.